src/orchard/namespace.clj

```diff
@@ -148,9 +151,30 @@
             io/resource ;; can return nil for Emacs backup files, for example
             read-ns-name)))
 
+(defn- find-cljc-files-efficiently
```

Comment:
I'd probably use a different name for this, as it now seems it looks for "cljc" files. Perhaps you can make the extensions params, so this can be a more generic function instead?

Reply:
We could use `find-clj+cljc-files-efficiently`?

The param is an acceptable option, OTOH making things generic would mean that we can't longer accurarately use the docstring for describing what we're gaining.

Reply:
I'd be OK with the name, although I don't quite understand what would be the issue with the docstring if the function was generic, though - it would be similar to the current one, but with a note that you have to specify the file extensions you're targetting.